### PR TITLE
feat(editor): Allow cmd+enter as option to run cell on macOS

### PR DESCRIPTION
**Pull request by SomeKittens**
- [x] I have read the [Contributor Guide](https://github.com/nteract/nteract/blob/master/CONTRIBUTING.md)
- [x] My PR title and commit messages are in [conventional-changelog-standard](https://github.com/nteract/nteract/blob/master/CONTRIBUTING.md#how-should-i-write-my-commit-messages-and-pr-titles) format.

Was playing around with a notebook on macOS, kept trying to run a code cell via `cmd + enter` rather than `ctrl + enter`, so I thought I'd open a PR for that.

### CODE REVIEW

File: src/notebook/components/notebook.js [hunk 1]
```diff
@@ -122,7 +122,13 @@ export class Notebook extends React.PureComponent {
       return;
     }
 
-    const shiftXORctrl = (e.shiftKey || e.ctrlKey) && !(e.shiftKey && e.ctrlKey);
+    let ctrlKeyPressed = e.ctrlKey;
+    // Allow cmd + enter (macOS) to operate like ctrl + enter
+    if (process.platform === 'darwin') {
+      ctrlKeyPressed = (e.metaKey || e.ctrlKey) && !(e.metaKey && e.ctrlKey);
```
Comment (rgbkrk) [inline src/notebook/components/notebook.js:128]: 👍

**Review by rgbkrk — APPROVED** [review]
Thanks for this, I'm really happy to have cmd-enter here. 😄

**Comment by SomeKittens** [comment]
Glad I could add it, quick fix, too.  Thanks!

**Comment by rgbkrk** [comment]
Welcome to the nteract party! 🎉

**Comment by rgbkrk** [comment]
Totally forgot about this! It's in last night's v0.2.0 release (finally shipped!)

**Pull request merged**